backend/internal/database/img.go

```diff
@@ -0,0 +1,75 @@
+package database
+
+import (
+	"bytes"
+	"context"
+	"image"
+
+	_ "image/jpeg"
+	_ "image/png"
+
+	"github.com/buckket/go-blurhash"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+)
+
+type ImgDatabase struct {
+	Conn *pgxpool.Pool
+}
+
+/*
+ * Gets an image from the image table given the id. If image exists, returns the binary data of the image
+ */
+func (db *ImgDatabase) GetImage(ctx context.Context, id string) (*[]byte, *string, error) {
```

Comment:
Any reason this returns pointers? If it's because of the potential for errors, it's usually more idiomatic to just return `nil, "", err` - by convention the first two values are always ignored if err != nil.